Block/StaticBlock.php

```diff
@@ -80,18 +74,27 @@ private function createPrismicDocument()
     /**
      * @return string
      * @throws ContextNotFoundException
-     * @throws DocumentNotFoundException
      */
     public function fetchDocumentView(): string
     {
-        if (! $this->fetchChildDocument()) {
+        try {
+            if (!$this->fetchChildDocument()) {
+                return '';
+            }
+        } catch (ApiNotEnabledException|ContextNotFoundException|DocumentNotFoundException|NoSuchEntityException $e) {
```

Comment:
Isnt't there an Exception that is used as an ancestor for all these Exception classes? If not, I'd just use `Exception` here, unless you want any other exception to actually throw an error.

Reply:
Changed to Exception